values.yaml

```diff
@@ -425,94 +425,129 @@ st2chatops:
+    # Whether to enable the arbiter
+    arbiter:
+      enabled: false
+
   resources: {}
 
 ##
-## RabbitMQ HA configuration (3rd party chart dependency)
+## RabbitMQ configuration (3rd party chart dependency)
 ##
 ## For values.yaml reference:
-## https://github.com/helm/charts/tree/master/stable/rabbitmq-ha
+## https://github.com/bitnami/charts/tree/master/bitnami/rabbitmq
 ##
-rabbitmq-ha:
+rabbitmq:
   # Change to `false` to disable in-cluster rabbitmq deployment.
   # Specify your external [messaging] connection parameters under st2.config
   enabled: true
+  
   # On unclean cluster restarts forceBoot is required to cleanup Mnesia tables (see: https://github.com/helm/charts/issues/13485)
   # Use it only if you prefer availability over integrity.
-  forceBoot: true
-  rabbitmqUsername: admin
-  # TODO: Use default random 24 character password, but need to fetch this string for use by downstream services
-  rabbitmqPassword: 9jS+w1u07NbHtZke1m+jW4Cj
-  # RabbitMQ Memory high watermark. See: http://www.rabbitmq.com/memory.html
+  clustering.forceBoot: true
+
+  # Authentication Details
+  auth:
+    username: admin
+  # TODO: Use default random 10 character password, but need to fetch this string for use by downstream services
+    password: 9jS+w1u07NbHtZke1m+jW4Cj
+    # Up to 255 character string, should be fixed so that re-deploying the chart does not fail (see: https://github.com/helm/charts/issues/12371)
+    # NB! It's highly recommended to change the default insecure rabbitmqErlangCookie value!
+    erlangCookie: 8MrqQdCQ6AQ8U3MacSubHE5RqkSfvNaRHzvxuFcG
+
+    # RabbitMQ Memory high watermark. See: http://www.rabbitmq.com/memory.html
   # Default values might not be enough for StackStorm deployment to work properly. We recommend to adjust these settings for you needs as well as enable Pod memory limits via "resources".
   #rabbitmqMemoryHighWatermark: 512MB
   #rabbitmqMemoryHighWatermarkType: absolute
-  # Up to 255 character string, should be fixed so that re-deploying the chart does not fail (see: https://github.com/helm/charts/issues/12371)
-  # NB! It's highly recommended to change the default insecure rabbitmqErlangCookie value!
-  rabbitmqErlangCookie: 8MrqQdCQ6AQ8U3MacSubHE5RqkSfvNaRHzvxuFcG
-  persistentVolume:
+  persistence:
     enabled: true
   # RabbitMQ application vhost, should match with 'ha' Queue Mirroring definition policy
-  rabbitmqVhost: "/"
-  definitions:
-    # Enable Queue Mirroring between nodes
-    # See https://www.rabbitmq.com/ha.html
-    policies: '{"vhost":"/","name":"ha","pattern":"", "definition":{"ha-mode":"all","ha-sync-mode":"automatic","ha-sync-batch-size":10}}'
+  ingress:
+    enabled: true
+    path: "/"
+
+  # Enable Queue Mirroring between nodes
+  # See https://www.rabbitmq.com/ha.html
+  extraConfiguration: '' 
+  #  policies = ['{"vhost":"/","name":"ha","pattern":"", "definition":{"ha-mode":"all","ha-sync-mode":"automatic","ha-sync-batch-size":10}}'']
```

Comment:
Mirroring RabbitMQ Queues are required for ST2 use, let's enable them.

Reply:
I was trying to figure out how to inject those in, I think the formatting is out on line 511. I will have another go at it.

Reply:
I have a change here that in theory should work but there is an issue with the bitnami chart, issue number https://github.com/bitnami/charts/issues/4635

Reply:
I'm happy to push the non working chunk in the values file up if you like anyway?

Reply:
I have it up and working, the policies are now loading into the sub chart.